docs/application/vstools/index.md

```diff
@@ -40,7 +40,7 @@ Visual Studio Tools for Tizen provides Tizen-specific tools to improve your prod
 
 - [Memory Profiler](tools/memory-profiler-user-manual.md)
 
-  This is a utility which collects memory profiling data and save them to a file, this part of the Memory Profiler runs on the target side. Also it analyzes the resulting data, the command-line one and the GUI-based one, these analyzing tools run on the host side.
+  This is a utility which collects memory profiling data and saves them to a file, this part of the Memory Profiler runs on the target side. Also, it analyzes the resulting data, the command-line one and the GUI-based one, these analyzing tools run on the host side.
```

Comment:
> "This is a utility which collects memory profiling data and saves them to a file, this part of the Memory Profiler runs on the target side."

**Suggestion**. Isn't it better to cut this sentence into two in the middle?
This is a utility which collects memory profiling data and saves them to a file. This part of the Memory Profiler runs on the target side.

> "Also, it analyzes the resulting data, the command-line one and the GUI-based one, these analyzing tools run on the host side."

**Suggestion**. How about changing this sentence like below?
 Also, it analyzes the resulting data, the command-line one and the GUI-based one, which run on the host side.

Reply:
Your suggestions are good but the current paragraph is not bad for the short summary.